packages/dev/core/src/Behaviors/Cameras/autoRotationBehavior.ts

```diff
@@ -23,7 +23,21 @@ export class AutoRotationBehavior implements Behavior<ArcRotateCamera> {
     private _idleRotationSpeed = 0.05;
     private _idleRotationWaitTime = 2000;
     private _idleRotationSpinupTime = 2000;
+    private _targetAlpha: number;
 
+    /**
+     * Sets the target aplha for Camera to rotate to
+     * @param targetAlpha
+     */
+    public setTargetAlpha(targetAlpha: number): void {

```

Comment:
instead of adding the setTargetAlpha function, you can use `set targetAlpha(...`, or simply make `targetAlpha` a public Nullable<number> instead of having both a getter and a setter.

Reply:
Thanks for the suggestion. I have implement this in the new commit @RaananW

Reply:
thanks a lot :-)